RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZ_vect.cc

```diff
@@ -63,13 +96,12 @@ DAClusterizerInZ_vect::track_t
 DAClusterizerInZ_vect::fill(const vector<reco::TransientTrack> & tracks) const {
 
   // prepare track data for clustering
-  LogDebug("DAClusterizerinZ_vectorized") << "input values"; 
   track_t tks;
   for (auto it = tracks.begin(); it!= tracks.end(); it++){
     if (!(*it).isValid()) continue;
     double t_pi=1.;
     double t_z = ((*it).stateAtBeamLine().trackStateAtPCA()).position().z();
-    if (std::abs(t_z) > 1000.) continue;
+    if (std::fabs(t_z) > 1000.) continue;
```

Comment:
It's a consequence of my personal ignorance. I had just learned the hard way that there is a difference between abs() and std::abs(). But for all I know now, it can/should indeed be reverted here.